internal/claude/parser.go

```
@@ -136,6 +173,10 @@ func LoadSessionsInTimeRange(startTime, endTime time.Time, projectFilter string,
 	if err != nil {
 		return nil, err
 	}
+	
```

golangci-lint (1.64.8)

File is not properly formatted

(gofmt)